<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtLvBaClass</class>
 <widget class="QWidget" name="QtLvBaClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>714</width>
    <height>494</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>714</width>
    <height>494</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>714</width>
    <height>494</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QtLvBa</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">QFrame{border-image: url(:/QtLvBa/137.bmp);}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>80</y>
        <width>102</width>
        <height>120</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{border-image:url(:/QtLvBa/00007[102x120x8BPP].png);}
QPushButton:hover{border-image: url(:/QtLvBa/00008[102x120x8BPP].png);}
QPushButton:pressed{border-image: url(:/QtLvBa/00008[102x120x8BPP].png);}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>270</y>
        <width>102</width>
        <height>120</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{border-image:url(:/QtLvBa/00004[102x120x8BPP].png);}
QPushButton:hover{border-image: url(:/QtLvBa/00003[102x120x8BPP].png);}
QPushButton:pressed{border-image: url(:/QtLvBa/00003[102x120x8BPP].png);}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>150</y>
        <width>102</width>
        <height>120</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{border-image:url(:/QtLvBa/00006[102x120x8BPP].png);}
QPushButton:hover{border-image: url(:/QtLvBa/00005[102x120x8BPP].png);}
QPushButton:pressed{border-image: url(:/QtLvBa/00005[102x120x8BPP].png);}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>330</y>
        <width>102</width>
        <height>120</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{border-image:url(:/QtLvBa/00010[102x120x8BPP].png);}
QPushButton:hover{border-image: url(:/QtLvBa/00009[102x120x8BPP].png);}
QPushButton:pressed{border-image: url(:/QtLvBa/00009[102x120x8BPP].png);}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>360</y>
        <width>102</width>
        <height>120</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{border-image:url(:/QtLvBa/00002[102x120x8BPP].png);}
QPushButton:hover{border-image: url(:/QtLvBa/00001[102x120x8BPP].png);}
QPushButton:pressed{border-image: url(:/QtLvBa/00001[102x120x8BPP].png);}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>420</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{border-image:url(:/QtLvBa/00011[51x20x8BPP].png);}
QPushButton:hover{border-image: url(:/QtLvBa/00012[51x20x8BPP].png);}
QPushButton:pressed{border-image: url(:/QtLvBa/00012[51x20x8BPP].png);}

</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>525</x>
        <y>420</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{border-image:url(:/QtLvBa/00014[51x20x8BPP].png);}
QPushButton:hover{border-image: url(:/QtLvBa/00014[51x20x8BPP].png);}
QPushButton:pressed{border-image: url(:/QtLvBa/00014[51x20x8BPP].png);}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_8">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>420</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{border-image:url(:/QtLvBa/00015[51x20x8BPP].png);}
QPushButton:hover{border-image: url(:/QtLvBa/00016[51x20x8BPP].png);}
QPushButton:pressed{border-image: url(:/QtLvBa/00016[51x20x8BPP].png);}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
